tests/microbenchmarks/skip_softmax/bench_skip_softmax_fmha_exe.py

```
@@ -0,0 +1,275 @@
+        cmd += ["-sliding-window-size", str(cfg.seq_len_kv)]
+    if cfg.num_heads_kv != cfg.num_heads_q:
+        cmd += ["-gqa", str(cfg.num_heads_kv)]
+    if threshold > 0:
+        cmd += ["-skip-softmax-threshold-scale-factor", f"{threshold:g}"]
+    if with_stat and threshold > 0:
+        cmd += ["-skip-softmax-stat"]
+    return cmd
+
+
+def _run_once(cmd: List[str]) -> Tuple[Optional[float], Optional[Tuple[int,
+                                                                      int]]]:
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)